satellite/gracefulexit/endpoint.go

```diff
@@ -922,3 +777,97 @@ func (endpoint *Endpoint) updatePointer(ctx context.Context, originalPointer *pb
 
 	return nil
 }
+
+func (endpoint *Endpoint) checkExitStatus(ctx context.Context, nodeID storj.NodeID) (*pb.SatelliteMessage, error) {
```

Comment:
let's add a comment above this that describes what this function does, but specifically clarifies that the message returned can be nil, and if it is not nil, it means that we should close the connection after sending